### PR TITLE
Modified Readme

### PR DESCRIPTION
**Pull request by liannah**

### CODE REVIEW

**Review by Arushi282 — CHANGES_REQUESTED** [review]
(1) Observed a couple of grammatical errors, which I will point out in separate comments

(2) In usage after explaining the conda environment, it is better to also have a dependencies section like in our proposal md, instead of linking to the environment yaml 

(3) I pretty sure you already know, but its just good to have a record of it, final model performance need to be discussed. 

Everything else looks good, thanks Lianna!

File: README.Rmd [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+- Authors: Arushi Ahuja, Karanpreet Kaur, Lianna Hovhannisyan, Taiwo Owoseni
+
+Using Logistic Regression to predict whether the customer will do credit default payment next month or not.
+
+## About
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+In this project, we built a classification model using Logistic Regression to predict if the credit account holders will make a default payment or not next month. The model was trained on features that hold information about the client's last 6 months bill and payment history, as well as several characteristics: age, marital status, gender, etc. Overall, we are more interested in minimizing Type | error (predicting not default payment, when in reality client made a default payment next month) and Type || error (predicting default payment, when in reality client did not do default next month), we are using primarily $f1$ as scoring metric. Our model performed fairly well....
```
Comment (Arushi282) [inline README.Rmd:16]: Changed the grammatical errors in capital:

In this project, we built a classification model using Logistic Regression to predict if [NO THE] credit account holders will make a default payment [NO OR NOT] next month. The model was trained on features that hold information about the client's last 6 months bill and payment history, as well as several OTHER characteristics SUCH AS: age, marital status, AND gender[NO ETC]. Overall, we are more interested in minimizing Type | error (predicting NO default payment, when in reality THE client made a default payment THE FOLLOWING MONTH), AS OPPOSED TO Type || error (predicting default payment, when in reality NO DEFAULT PAYMENT WAS MADE BY THE CLIENT), we are using [NO PRIMARILY] $f1$ as OUR PRIMARY scoring metric. Our model performed fairly well....

**Comment by liannah** [comment]
> (1) Observed a couple of grammatical errors, which I will point out in separate comments
> 
> (2) In usage after explaining the conda environment, it is better to also have a dependencies section like in our proposal md, instead of linking to the environment yaml
> 
> (3) I pretty sure you already know, but its just good to have a record of it, final model performance need to be discussed.
> 
> Everything else looks good, thanks Lianna!

Thank you for reviewing for the first point. 

For the second, I will modify and add the dependencies after the file is ready.

For the third, yeah I will need to add it after final version is ready.

**Comment by liannah** [comment]
> > (1) Observed a couple of grammatical errors, which I will point out in separate comments
> > (2) In usage after explaining the conda environment, it is better to also have a dependencies section like in our proposal md, instead of linking to the environment yaml
> > (3) I pretty sure you already know, but its just good to have a record of it, final model performance need to be discussed.
> > Everything else looks good, thanks Lianna!
> 
> Thank you for reviewing for the first point.
> 
> For the second, I will modify and add the dependencies after the file is ready.
> 
> For the third, yeah I will need to add it after final version is ready.

I have added the changes, the dependecies will be updated when the file is ready. If no other comment, I think we can merge this for now, and I will make separate PR for the final version. @Arushi282

**Comment by thayeylolu** [comment]
yeap. looks good

**Review by Arushi282 — APPROVED** [review]

**Pull request merged**